oaknational/Oak-Web-Application
# Feat/newsletter dropdown

## Description

- adds ChevronDown component
- updates fontsizes/colors to be in line with designs
- forwards ref for compatibility react-hook-forms
- adds react-aria SSRProvider to ensure server/clientside ids are consistent
- changes DropdownSelect api onChange to be `(e: ChangeEvent) => void` for react-hook-forms
- removes focus outline (so that browser default is used)
- adjusts stories/tests
- adds DropdownSelect to Newsletter form

## Issue(s)

Fixes #292 

## How to test

1. Go to {preview deployment}
2. Attempt form submission
3. User type dropdown should work as expected

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/177334650-f8a9a515-309e-4d6c-974d-fb11d718f626.png)


## Checklist
